src/logic/AppLogger.ts

```diff
@@ -0,0 +1,30 @@
+//config.js
+import { logger, configLoggerType } from 'react-native-logs';
+import { rnFsFileAsync } from 'react-native-logs/dist/transports/rnFsFileAsync';
+import { ansiColorConsoleSync } from 'react-native-logs/dist/transports/ansiColorConsoleSync';
+import * as RNFS from 'react-native-fs';
+
+console.log('starting global logger');
+const nowTimeStamp: string = new Date().toISOString().replace(/\.|:/g, '-');
+const logDirectory: string = `${RNFS.ExternalDirectoryPath}/app-logs`;
+RNFS.mkdir(logDirectory);
```

Comment:
This returns a promise and we could potentially have this failing or taking too long, which could crash the app.

However, I think for now, we can assume this is quick enough and that it won't be failing in most cases. If a crash is experienced in any case, it will be refactored in the future, or just have a TODO for later.